### PR TITLE
Update project configuration

### PR DESCRIPTION
**Pull request by MiranDMC**
Updated framework and language version.
Fixed SDK includes.
Configured post build events and debug settings.
Added setup_env.bat and SETUP.md

### CODE REVIEW

File: CLEO4.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27004.2002
+# Visual Studio Version 17
```
Comment (x87) [inline CLEO4.sln:3]: is it VS 2022?

File: CLEO4.vcxproj [hunk 2]
```diff
@@ -62,46 +62,47 @@
...
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug GTASA|Win32'">
-    <OutDir>$(SolutionDir)output\Debug\</OutDir>
-    <IntDir>$(SolutionDir)output\.obj\Debug\</IntDir>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
+    <OutDir>$(SolutionDir)output\$(Configuration)\</OutDir>
+    <IntDir>$(SolutionDir)output\.obj\$(Configuration)\</IntDir>
     <TargetName>CLEO</TargetName>
     <TargetExt>.asi</TargetExt>
-    <IncludePath>$(VC_IncludePath);$(WindowsSdk_71A_IncludePath);E:\Documents\Para GTA\plugin-sdk-master\shared</IncludePath>
+    <IncludePath>$(PLUGIN_SDK_DIR)\shared\;$(PLUGIN_SDK_DIR)\shared\game\;$(IncludePath)</IncludePath>
```
Comment (x87) [inline CLEO4.vcxproj:103]: would it solve the issue with plugin sdk file import (CGame::bMissionPackGam)

File: CLEO4.sln (hunk 1 above)
Comment (MiranDMC) [reply CLEO4.sln:3]: Yes

File: CLEO4.vcxproj (hunk 2 above)
Comment (MiranDMC) [reply CLEO4.vcxproj:103]: Hmm, I don't think this will help.

**Pull request merged**